src/ISADotNet.XLSX/ISADotNet.XLSX.fsproj

```diff
@@ -17,11 +19,28 @@
     <Compile Include="InvestigationFile\Study.fs" />
     <Compile Include="InvestigationFile\OntologySourceReference.fs" />
     <Compile Include="InvestigationFile\Investigation.fs" />
-    <None Include="paket.references" />
-    <None Include="paket.template" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FSharpSpreadsheetML" Version="0.0.3" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\ISADotnet\ISADotNet.fsproj" />
   </ItemGroup>
-  <Import Project="..\..\.paket\Paket.Restore.targets" />
-</Project>
\ No newline at end of file
+  
+  <PropertyGroup>
+    <Authors>nfdi4plants, Lukas Weil, Kevin Frey</Authors>
```

Comment:
`nfdi4plants `is not really an `Author`, is it? I think if there is an `Owners` field, it should be moved there, together with `HLWeil`

Reply:
Owners is deprecated and authors should be used instead: https://docs.microsoft.com/de-de/nuget/reference/nuspec#owners